src/components/Form.js

```diff
@@ -135,8 +135,35 @@ function Form(props) {
 
             // Validate the input for html tags. It should supercede any other error
             _.each(trimmedStringValues, (inputValue, inputID) => {
+
+                const workspaceRegex = inputValue.search(CONST.VALIDATE_FOR_HTML_TAG_REGEX);
+                const matches = inputValue.match(CONST.VALIDATE_FOR_HTML_TAG_REGEX);
+
+                let isMatch = CONST.WHITELISTED_TAGS.some(r => r.test(inputValue));
+                const leadingSpaceRegex = inputValue.search(CONST.VALIDATE_FOR_LEADINGSPACES_HTML_TAG_REGEX);
+
                 // Return early if there is no value OR the value is not a string OR there are no HTML characters
-                if (!inputValue || !_.isString(inputValue) || inputValue.search(CONST.VALIDATE_FOR_HTML_TAG_REGEX) === -1) {
+                if (!inputValue || !_.isString(inputValue) || (leadingSpaceRegex === -1 && workspaceRegex === -1) ) {
+                    return;
+                }
+
+                // check for any matches that the original regex (workspaceRegex) matched
+                if(matches)
+                {
+                    // original regex does not catch leading spaces before tagnames 
+                    // need to check if any invalid input is matched along valid ignoreRegxList
```

Comment:
We don't have a `ignoreRegxList` variable, did you mean `WHITELISTED_TAGS` here?